jax/ops/scatter.py

```diff
@@ -334,4 +341,19 @@ def segment_sum(data, segment_ids, num_segments=None,
 
   out = jnp.zeros((num_segments,) + data.shape[1:], dtype=data.dtype)
   segment_ids = jnp.mod(segment_ids, num_segments)
-  return index_add(out, segment_ids, data, indices_are_sorted, unique_indices)
+
+  num_buckets = 1 if bucket_size is None \
+                  else -(-segment_ids.size // bucket_size)  # ceil_division
```

Comment:
I wonder if it's worth adding a helper function (possibly to some common place in JAX) called `ceil_of_ratio`.

TF and XLA have a similar utility in C++.

Reply:
Done.